whats_on/app/views/components/segmented-control/segmented-control.njk

```diff
@@ -38,7 +38,9 @@
             {s:3} | spacingClasses({padding: ['top', 'bottom']}),
             'border-top-width-1' if loop.first
           ].join(' ') }} segmented-control__drawer-item border-bottom-width-1 border-color-smoke">
-            <a href="{{ item.url }}" class="js-segmented-control__drawer-link segmented-control__drawer-link block plain-link {{ 'js-trap-reverse-start' if loop.last }}  {{'js-proxy-tablink' if data.isTabControl }}">
+            <a data-track-link="{{ {category: 'component', action: 'whats-on-daterange-picker:click', label: 'title:' + item.title} | dump }}"
```

Comment:
Shouldn't these be `data-track-event`?

Reply:
Don't know what you're talking about (rebase might)